src/app/tests/suites/TestModeSelectCluster.yaml

```diff
@@ -88,3 +93,23 @@ tests:
                 value: 2
       response:
           error: CONSTRAINT_ERROR
+
+    - label: "Change OnMode"
+      command: "writeAttribute"
+      attribute: "OnMode"
+      arguments:
+        value: 7
+
+    - label: "Toggle OnOff"
+      cluster: "On/Off"
+      command: "off"
+
+    - label: "Toggle OnOff"
+      cluster: "On/Off"
+      command: "on"
+
+    - label: "Verify Current Mode Change"
+      command: "readAttribute"
+      attribute: "CurrentMode"
+      response:
+        value: 7
```

Comment:
Shouldn't there be a test where we write null to OnMode and check how it behaves?